experimental/runtimes/src/thread_manager.rs

```diff
@@ -65,3 +66,10 @@ impl ThreadManagerBuilder {
         }
     }
 }
+
+/// This assumes that we get a peak performance at 48 parallelism - in future if we can scale up
+/// our code for higher parallelism, we can increase this number.
+static OPTIMAL_MAX_PARALLELISM: usize = 48;
+pub fn optimal_min_parallelism(num_tasks: usize, min_threshold: usize) -> usize {
```

Comment:
"parallelism" in "max parallelism" and "min parallelism" obviously mean different things?

Reply:
Do you have a better suggestion for naming :)

Reply:
`optimal_min_len()` to be brutal?